app/src/main/java/org/kiwix/kiwixmobile/webserver/ZimHostFragment.kt

```diff
@@ -215,14 +231,16 @@ class ZimHostFragment : BaseFragment(), ZimHostCallbacks, ZimHostContract.View {
   }
 
   private fun select(bookOnDisk: BooksOnDiskListItem.BookOnDisk) {
-    val booksList: List<BooksOnDiskListItem> = booksAdapter.items.map {
+    allBooks = booksAdapter.items.map {
```

Comment:
allBooks only being set when you select something is odd as it is not set in `addBooks`, which is what gets called by the Presenter

Reply:
if we select something then it saves the `SharedPreference` then onResume 
it calls this 
```
     presenter.loadBooks(sharedPreferenceUtil.allBooksToHost)
```